__tests__/app.test.js

```diff
@@ -248,4 +248,87 @@ describe('app', () => {
                 });
         });
     })
+    describe("PATCH /api/artices/:articled_id", () => {
+        test("article up-voted by 1 status 201 response is updated article", () => {
+            const updatedArticle = {
+                inc_votes: 1,
+            };
+            return request(app)
+                .patch("/api/articles/1")
+                .send(updatedArticle)
+                .expect(201)
```

Comment:
status code of 201 means something has been created. As the article is already created here and is being updated, a status code of 200 would be more suitable